corehq/tabs/tabclasses.py

```diff
@@ -1175,19 +1175,24 @@ class ProjectUsersTab(UITab):
     @property
     def _is_viewable(self):
         return self.domain and (self.couch_user.can_edit_commcare_users() or
+                                self.couch_user.can_view_commcare_users() or
```

Comment:
Throughout this commit, the various `can_view_foo or can_edit_foo` could all be simplified to `can_view_foo`, right?I kind of like how explicit it is to check both view and edit, but it's also verbose.

Reply:
true, but I really wanted to be explicit with permissions